src/main/fc/fc_msp.c

```diff
@@ -485,6 +485,22 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
 #endif
         break;
 
+    case MSP2_INAV_OPTICAL_FLOW:
+#ifdef USE_OPTICAL_FLOW
+        sbufWriteU8(dst, opflow.rawQuality);
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.flowRate[X]));
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.flowRate[Y]));
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.bodyRate[X]));
+        sbufWriteU16(dst, RADIANS_TO_DEGREES(opflow.bodyRate[Y]));
+#else
+        sbufWriteU8(dst, opflow.rawQuality);
```

Comment:
I can't see a difference between the `#if` and the `#else` clause.

Reply:
Oops!